README.md

```diff
@@ -50,22 +50,24 @@ Supported Ruby Interpreters
 ## Installation
 
 ```bash
-    gem install webmock
+gem install webmock
 ```
 or alternatively:
 
 ```ruby
-    # add to your Gemfile
-    group :test do
-        gem "webmock"
-    end
+# add to your Gemfile
+group :test do
+  gem "webmock"
+end
 ```
 
 ### or to install the latest development version from github master
 
-    git clone http://github.com/bblimke/webmock.git
-    cd webmock
-    rake install
+```bash
+git clone http://github.com/bblimke/webmock.git
+cd webmock
+rake install
+```
```

Comment:
Granted this one is more of replacing the alternate 4-space indent codeblock style, but seeing as the rest of the doc was using backticks this seemed to make sense.